client/src/main/python/ss-scale-disk.py

```diff
@@ -51,6 +51,8 @@ def _validate_and_set_scale_options(self):
 
     def doWork(self):
         super(MainProgram, self).doWork()
+        action = self.options.attach_gb and 'attach' or 'detach'
+        print('Requested %sment of extra disk on %s' % (action, self.node_name))
```

Comment:
After checking with @loomis. The call is asynchronous, os it's OK to leave this wording.